### ISSUE TITLE
Crashing on Mac

### ISSUE THREAD

**Issue by dhantasmic** [Facepunch/garrysmod-issues#3404]
This crash issue started right after the update this morning.  Might be a Windows issue too, but the two people I've had report it were on Mac, so it might be contained to Mac?  Really not sure what is causing it. 

This was on the Prop Hunt gamemode, but that may have no relevance.  

Crash: https://pastebin.com/Z8s24vSs

**Comment by dhantasmic** [comment]
Any chances of this getting fixed?  Even if someone has an idea of what may be causing this, maybe I can disable specific features in my server to help players stop crashing.

**Comment by Xorkle** [comment]
Hi, I'm getting frequent crashes too :(

**Comment by dhantasmic** [comment]
I seem to have narrowed it down to something with the SMG grenades and people dying at the same time.  It doesn't happen consistently though.  I've just disabled those grenades for now and no crashes have been happening.

**Comment by zusezuse** [comment]
The same issue, I got DarkRP to run for 30 minutes at a time, and then it eventually stopped working altogether. I can't even join games without a crash anymore.

**Comment by rudeustudeus** [comment]
Also getting intermittent crashing. Mostly occurs after reconnecting or dying.

**Comment by willox** [comment]
Is anybody able to provide another crash dump like in the original comment?

**Comment by dhantasmic** [comment]
For anyone who doesn't know how to get a crash dump, this page provides Windows and OSX instructions: https://wiki.garrysmod.com/page/Crash_Reporting

I'd love to get this fixed because right now it's impossible to play any gamemodes with the constant crashing.

**Comment by robotboy655** [comment]
Potentially related to https://github.com/Facepunch/garrysmod-issues/issues/3401

**Comment by FR-Numbor** [comment]
I was able to get the crash report of one of my latest crashes, hopefully it helps fixing this problem.

Crash Report: https://pastebin.com/aGn0tc55

**Comment by andbass** [comment]
I am also encountering this issue. Looking at the stack trace, seems to be a problem with the sentence mixer (CSentenceMixer::CSentenceMixer). My stack trace seems fairly similar to dhantasmic's.

Crash Report: [https://pastebin.com/wVEkQX3r](url)

**Comment by dhantasmic** [comment]
More crash reports from someone on the forums: https://gmod.facepunch.com/f/gmodhelp/rwcf/Frequent-crashing-on-macOS/

https://pastebin.com/6uGGTuKw
https://pastebin.com/L87sF6Zz

**Comment by NeedlessMemeing** [comment]
I also had a similar issue, I suicided and it crashed to the desktop. I was using a custom playermodel, but that rarely causes any issues.
https://pastebin.com/afkeKCPX

**Comment by willox** [comment]
Could you see if this still happens on the GMod dev branch? If it does, please post logs from there too.

![image](https://user-images.githubusercontent.com/1499676/37284765-b86c22f0-25f4-11e8-8f50-4604fda2a159.png)
![image](https://user-images.githubusercontent.com/1499676/37284769-bb2d6e90-25f4-11e8-97df-7d49acf7cc14.png)

**Comment by Jake3231** [comment]
@willox I have been having the same issue on my Mac since the recent updates. I have tried to completely reinstall the game as well as remove quite a few addons. Removing addons slightly postpones the cash, but the game always quits after about a minute of light gameplay.

I have attached two different crash reports from the dev branch.

[GmodCrash1.txt](https://drive.google.com/open?id=1Z4-6DiJH_IVxMEcQ8Z4jm0g1mjHjnAhz)
[GmodCrash2.txt](https://drive.google.com/open?id=134dppox3PNPL3NllRQTHWISeOKrBl8BF)

MacBook Pro 13" Early 2015
2.7 Ghz Intel Core i5
8GB RAM @ 1876 MHz
macOS 10.13.3
